.github/CODEOWNERS

```diff
@@ -17,6 +17,11 @@
 # Shayna Finocchiaro (shaynafinocchiaro)
 # Logan Jones (lj-software)
```

Comment:
Please remove Logan and Michael from the code owners list. Thank you!

Reply:
Addressed